src/newcharacter.cpp

```diff
@@ -2650,6 +2650,9 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
             if( !MAP_SHARING::isSharing() ) { // Don't allow random names when sharing maps. We don't need to check at the top as you won't be able to edit the name
                 you.pick_name();
             }
+            you.set_base_age( rng( 16, 55 ) );
+            you.set_base_height( rng( 145, 200 ) );
+            you.male = one_in( 2 );
```

Comment:
You should either rename the "PICK_RANDOM_NAME" action or put this in a different action.